client/gutenberg/editor/components/header/header-toolbar/index.js

```diff
@@ -50,8 +51,17 @@ function HeaderToolbar( {
 	// GUTENLYPSO END
 } ) {
 	const onCloseButtonClick = () => {
-		notices.forEach( ( { id } ) => removeNotice( id ) );
-		closeEditor();
+		// GUTENLYPSO START
+		const proceed =
+			isDirty && typeof window !== 'undefined'
+				? window.confirm( __( 'Changes that you made may not be saved.' ) )
```

Comment:
Not a big deal really, but maybe we can use the same text that's in Gutenberg:

> You have unsaved changes. If you proceed, they will be lost.

https://github.com/WordPress/gutenberg/blob/d95c7d61d8ca284394a5aa77e9f74f9b3aaea24e/packages/editor/src/components/unsaved-changes-warning/index.js#L33

Reply:
+1 for this change

Reply:
Updated in 11d4afbcf8d6af928209ab34e4dae289d39c1f61.